tests/testthat/test-geo.R

```diff
@@ -0,0 +1,89 @@
+context("Various geo functions")
+
+with_mock_crunch({
+    ds <- loadDataset("test ds")
+    geo_data <- geo(ds$location)
+    test_that("geo getter", {
+        expect_equal(geo_data$feature_key, "properties.location")
+        expect_equal(geo_data$match_field, "name")
+        expect_equal(geo_data$geodatum, "https://app.crunch.io/api/geodata/8684c65ff11c4cc3b945c0cf1c9b2a7f/")
+    })
+    
+    test_that("geo error", {
+        # if there is no geography on a variable, geo() returns null.
```

Comment:
Make this and the other comments like this be the message in `test_that` so that they show up in the test output